components/registry-facade/pkg/registry/manifest_test.go

```diff
@@ -99,6 +100,12 @@ func TestDownloadManifest(t *testing.T) {
 			}
 			_, _, err = DownloadManifest(context.Background(), fetcher, desc, WithStore(test.Store))
 			if err != nil {
+				// Devspaces-specific start
+				// * if err includes "DS: ", it's a digest mismatch and we can ignore it
```

Comment:
Will this always be the case though ? Devspaces specific for now only means digest mismatch, but as we keep adding more extensions and capabilities it might mean something else. Can we make the error comparison filter more specific so as to not run into false positives later down the road

Reply:
The initial idea was that in case the test cases are failing due to a Devspaces specific error, we will have throw an error of format "DS: ...". That way we don't have to check for every possible error that we need to bypass in test suites. 

But now I agree that this is a slippery road for false positives.

Reply:
We don't need to handle errors here anymore, so I am removing this altogether